datadog/resource_datadog_logs_pipelineorder.go

```diff
@@ -0,0 +1,78 @@
+package datadog
```

Comment:
Let's also rename this file to be `pipeline_order` to match the suggested new name of the resource.